addons/mail/static/src/models/autocomplete_input_view/autocomplete_input_view.js

```diff
@@ -0,0 +1,226 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'AutocompleteInputView',
+    identifyingFields: [[
+        'chatWindowOwnerAsNewMessageFormInputOwner',
+        'discussAsChannelMobileAddItemHeaderOwner',
+        'discussAsChatMobileAddItemHeaderOwner',
+        'discussSidebarCategoryAsAddingItemInputViewOwner',
+    ]],
+    lifecycleHooks: {
+        _willDelete() {
+            if (this.discussAsChannelMobileAddItemHeaderOwner) {
+                this.discussAsChannelMobileAddItemHeaderOwner.clearIsAddingItem();
+            }
+        },
+    },
+    recordMethods: {
+        /**
+         * @param {MouseEvent} ev
+         */
+        onBlur(ev) {
+            if (!this.exists()) {
+                return;
+            }
+            this.delete();
```

Comment:
This behavior was not present before for chat window (it didn't pass `onHide`), so it leads to a strange state.

Reply:
Done